### PR TITLE
Add just test-stream to run a test with live output

### PR DESCRIPTION
**Pull request by npaun**
I find it a lot more convenient than rooting around in the test log

### CODE REVIEW

File: justfile [hunk 1]
```diff
@@ -28,6 +28,9 @@ test *args="//...":
 test-asan *args="//...":
   just test {{args}} --config=asan
 
+test-stream args:
```
Comment (anonrig) [inline justfile:31]: Can you add an example argument as a comment to here? Should we have `//...` as the default value?

Comment (anonrig) [inline justfile:31]: Maybe lets call it `stream-tests` and add a shortcut for `st` so we can execute it using `just st` 
```suggestion
stream-tests args:
```

Comment (npaun) [reply justfile:31]: I think that would be too slow and too much output - I think it's best when running a single test.

Comment (npaun) [reply justfile:31]: Nice!

Comment (anonrig) [reply justfile:31]: Would you mind adding a comment for running an example test?

**Review by anonrig — APPROVED** [review]

**Pull request merged**